dot/parachain/backing/integration_test.go

```diff
@@ -44,6 +44,8 @@ func initBackingAndOverseerMock(t *testing.T) (*backing.CandidateBacking, *overs
 
 	backing.Keystore = keystore.NewBasicKeystore("test", crypto.Sr25519Type)
 
+	sr25519.NewPublicKey(backing.Keystore.Keypairs()[0].Public().Encode())
```

Comment:
is this code needed?

Reply:
haha no, I was trying to test something for the keypair stuff and forgot to remove it. will remove